Set user to prevent failures when not using a gateway

priteau opened the pull request:
When cluster_fixed_ip is set, we fetch the `user` attribute from cluster_groups to determine how to connect to the login node.

brtkwr commented on group_vars/openstack/slurm:
```diff
@@ -9,6 +9,7 @@ slurm_login:
   root_volume_size: 0
   node_resource: "Cluster::Node" 
   nodenet_resource: "Cluster::NodeNet1"
+  user: centos
```
can you please change this to `{{ centos_deploy_user }}`

priteau replied:
Changed to `user: "{{ cluster_deploy_user }}"`

brtkwr commented on group_vars/openstack/slurm:
```diff
@@ -17,6 +18,7 @@ slurm_compute:
   root_volume_size: 0
   node_resource: "Cluster::Node" 
   nodenet_resource: "Cluster::NodeNet1"
+  user: centos
```
can you please change this to {{ centos_deploy_user }}

priteau replied:
Changed to `user: "{{ cluster_deploy_user }}"`

priteau committed:
Set user to prevent failures when not using a gateway

When cluster_fixed_ip is set, we fetch the `user` attribute from
cluster_groups to determine how to connect to the login node.
